swagger.json

```diff
@@ -6855,15 +6855,26 @@
                             "items": {
                                 "type": "object",
                                 "properties": {
-                                    "tagName": {
-                                        "type": "string",
-                                        "description": "The name of the tag to grant this user access to.",
-                                        "example": "North Carolina Distribution"
+                                    "tagId": {
+                                        "type": "number",
+                                        "format": "int64",
+                                        "description": "The id of the tag to grant this user access to.",
+                                        "example": "123"
                                     },
-                                    "role" : {
+                                    "roleId" : {
+                                        "type": "number",
+                                        "description": "The id of the standard role type",
+                                        "example": "123f"
+                                    },
+                                    "roleName" : {
                                         "type": "string",
-                                        "description": "The name of the role the user is assigned to for this tag. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
+                                        "description": "The name of the standard role type (case insensitive)",
```

Comment:
Per our in-person discussion, could we define the enum of possible values for `roleName`, and drop `roleId`?

Reply:
yup, will update this PR accordingly (i was also pretty lax on description until we talked :P)

Reply:
added enum roles